docs/manual.md

```diff
@@ -851,14 +950,17 @@ SendAck ==
     ...
 ```
 
-As you can see, we have to annotate only those records that do not have all
-four fields of `MT`. As soon as we have added the annotations, the type checker
-stopped complaining and let the model checker to run. The annotated code can be
-found in
+As you can see, we have to annotate only those records that are subtypes of
+`MT`: since `ackno` has exactly the type `MT`, Apalache's type inference doesn't
+need any help. As soon as we have added the annotations, the type checker stops
+complaining and lets the model checker run. The annotated code can be found in
 [`test/tla/HandshakeWithTypes.tla`](../test/tla/HandshakeWithTypes.tla).
 
-Type annotations can be also applied to sets of records. For example:
+Type annotations can also be applied to sets of records. For example:
 
+<!-- TODO(shonfeder): I read this as saying that the record type on the lhs is a
```

Comment:
@konnov or @andrey-kuprianov for my own edification, I'm hoping to get a bit of clarification on this point.

Reply:
Let's discuss this later with @konnov ;)

Reply:
Sounds good. I'll make a personal note here and ask at some future time.

Reply:
Hmmm. How can I see this comment in a diff?

Reply:
Oh, sorry, I dropped it from the PR already, you can see the comment by looking at https://github.com/informalsystems/apalache/pull/155/commits/e956b32a736604b456ac460d2bb2f6807a5fb3c3#diff-4b7e43801d0bf529f907fe63667db023R922

Reply:
To elaborate a bit: 

This doesn't make sense to me

```tla
[syn |-> BOOLEAN, ack |-> BOOLEAN, seqno |-> Int] <: {MT}
```

For the same reason this doesn't make sense to me:

```tla
3 <: {Nat}
```

Since we earlier define the type alias

```tla
 MT == [syn |-> BOOLEAN, ack |-> BOOLEAN, seqno |-> Int, ackno |-> Int]
```

I'd think it should be

```tla
{[syn |-> BOOLEAN, ack |-> BOOLEAN, seqno |-> Int]} <: {MT}
```

I.e., the set containing a record that is a (sub)type of `MT` is has the type of a set of `MT`. But I think I may be missing something about the notation here.